src/client/Microsoft.Identity.Client/Internal/Broker/BrokerSilentRequest.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.UI;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Internal.Broker
+{
+    internal class BrokerSilentRequest
+    {
+        private Dictionary<string, string> _brokerPayload = new Dictionary<string, string>();
+        private IBroker Broker { get; }
+        private readonly AcquireTokenSilentParameters _silentParameters;
+        private readonly AuthenticationRequestParameters _authenticationRequestParameters;
+        private readonly IServiceBundle _serviceBundle;
+
+        internal BrokerSilentRequest(
+            AuthenticationRequestParameters authenticationRequestParameters,
+            AcquireTokenSilentParameters acquireTokenSilentParameters,
+            IServiceBundle serviceBundle,
+            IBroker broker)
+        {
+            _authenticationRequestParameters = authenticationRequestParameters;
+            _silentParameters = acquireTokenSilentParameters;
+            _serviceBundle = serviceBundle;
+            Broker = broker;
+        }
+
+        public async Task<MsalTokenResponse> SendTokenRequestToBrokerAsync()
+        {
+            //TODO: call can invoke broker here
+            _authenticationRequestParameters.RequestContext.Logger.Info(LogMessages.CanInvokeBrokerAcquireTokenWithBroker);
```

Comment:
nit: log messages are a way to document the code. but not when they are hidden behind constants...